plotters-backend/src/text.rs

```diff
@@ -62,6 +62,9 @@ pub mod text_anchor {
     /// The horizontal position of the anchor point relative to the text.
     #[derive(Clone, Copy)]
     pub enum HPos {
+        /// Default value (Left), except chart axes that might provide a different pos.
+        /// Use another variant to override the default positionning
+        Default,
```

Comment:
I think it makes sense to derive `Default` for both HPos and VPos since they already have a `Default` variant.

Reply:
I am not sure I understood what you meant. Did you mean something like this ?
```diff
diff --git a/plotters-backend/src/text.rs b/plotters-backend/src/text.rs
index bcf3a7a..4acd931 100644
--- a/plotters-backend/src/text.rs
+++ b/plotters-backend/src/text.rs
@@ -60,10 +60,11 @@ impl<'a> From<&'a str> for FontFamily<'a> {
 /// ```
 pub mod text_anchor {
     /// The horizontal position of the anchor point relative to the text.
-    #[derive(Clone, Copy)]
+    #[derive(Default, Clone, Copy)]
     pub enum HPos {
         /// Default value (Left), except chart axes that might provide a different pos.
         /// Use another variant to override the default positionning
+        #[default]
         Default,
         /// Anchor point is on the left side of the text
         Left,
@@ -74,10 +75,11 @@ pub mod text_anchor {
     }

     /// The vertical position of the anchor point relative to the text.
-    #[derive(Clone, Copy)]
+    #[derive(Default, Clone, Copy)]
     pub enum VPos {
         /// Default value (Top), except chart axes that might provide a different pos
         /// Use another variant to override the default positionning
+        #[default]
         Default,
         /// Anchor point is on the top of the text
         Top,
@@ -88,7 +90,7 @@ pub mod text_anchor {
     }

     /// The text anchor position.
-    #[derive(Clone, Copy)]
+    #[derive(Default, Clone, Copy)]
     pub struct Pos {
         /// The horizontal position of the anchor point
         pub h_pos: HPos,
@@ -111,22 +113,6 @@ pub mod text_anchor {
         pub fn new(h_pos: HPos, v_pos: VPos) -> Self {
             Pos { h_pos, v_pos }
         }
-
-        /// Create a default text anchor position (top left).
-        ///
-        /// - **returns** The default text anchor position
-        ///
-        /// ```rust
-        /// use plotters_backend::text_anchor::{Pos, HPos, VPos};
-        ///
-        /// let pos = Pos::default();
-        /// ```
-        pub fn default() -> Self {
-            Pos {
-                h_pos: HPos::Default,
-                v_pos: VPos::Default,
-            }
-        }
     }
 }
```